releasenotes/notes/qiskit-algorithms-incorporated-421554a4ff547d0d.yaml

```diff
@@ -0,0 +1,59 @@
+---
+prelude: >
+    This release includes the migration of a subset of Qiskit Algorithms features to Qiskit Machine Learning.
+    This ensures continued enhancement of essential features for Qiskit Machine Learning following the end
+    of official support for Qiskit Algorithms. Therefore, Qiskit Machine Learning will no longer depend on
+    Qiskit Algorithms, possibly introducing breaking changes in import structures. Some quick-fixes are
+    described below.
+
+features:
+  - |
+    Migrated essential Qiskit Algorithms features to Qiskit Machine Learning:
+    - `qiskit_algorithms/gradients` -> `qiskit_machine_learning/gradients`
+    - `qiskit_algorithms/optimizers` -> `qiskit_machine_learning/optimizers`
+    - `qiskit_algorithms/state_fidelities` -> `qiskit_machine_learning/state_fidelities`
+    - Partial merge of `qiskit_algorithms/utils` with `qiskit_machine_learning/utils`
+  - |
+    Unit tests migrated accordingly:
+    - `qiskit_algorithms/tests/gradients` -> `qiskit_machine_learning/tests/gradients`
+    - `qiskit_algorithms/tests/optimizers` -> `qiskit_machine_learning/tests/optimizers`
+    - `qiskit_algorithms/tests/state_fidelities` -> `qiskit_machine_learning/tests/state_fidelities`
+    - Partial merge of `qiskit_algorithms/tests/utils` with `qiskit_machine_learning/tests/utils`
+
+issues:
+  - |
+    Incorporating Qiskit Algorithms will facilitate the upgrade to newer versions of Qiskit and
+    Qiskit Runtime. In particular, the following issues, still outstanding, will be tackled in the
+    next release:
+    - Support V2 primitives (#742)
+    - Add support for EstimatorV2 to run circuits over hardware (#810)
+  - |
+    Compliance with ISA standards:
+    - ISA circuit support for latest Runtime (#786)
+    - Sampler fails to run FidelityKernel even if circuits are transpiled (#165)
+
+upgrade:
+  - |
+    The merge of some of the features of Qiskit Algorithms into Qiskit Machine Learning might lead to breaking changes.
+    For this reason, caution is advised when updating to version 0.8 during critical production stages in a project.
+    Users must update their imports and code references in code that uses Qiskit Machine Leaning and Algorithms:
+    - Change `qiskit_algorithms/gradients` to `qiskit_machine_learning/gradients`
+    - Change `qiskit_algorithms/optimizers` to `qiskit_machine_learning/optimizers`
+    - Change `qiskit_algorithms/state_fidelities` to `qiskit_machine_learning/state_fidelities`
```

Comment:
As imports and sphinx refs use `.` would it be preferable here to show them that way e.g.

> Change `qiskit_algorithms.gradients` to `qiskit_machine_learning.gradients`

Reply:
Changed using `.`